test/res.redirect.js

```diff
@@ -104,7 +104,7 @@ describe('res', function(){
       .set('Accept', 'text/html')
       .expect('Content-Type', /html/)
       .expect('Location', 'http://google.com')
-      .expect(302, '<p>' + http.STATUS_CODES[302] + '. Redirecting to <a href="http://google.com">http://google.com</a></p>', done);
+      .expect(302, '<p>' + statuses[302] + '. Redirecting to <a href="http://google.com">http://google.com</a></p>', done)
```

Comment:
Just hard-code the literal text here. It was dynamic because `http` was not consistent between major versions, but `statuses` will be consistent.